### PR TITLE
Created current activity component for heads up display

### PR DESCRIPTION
**Pull request by ngwatso**
# Description

This feature was created to display the current activity to the user in the heads up display

-Component will calculate the current day in a 14 day rotation
-Based on the current day, component will select the current activity
-Activity will change daily at midnight, rendering onto the display

 -Trello:
     https://trello.com/c/JYNECJFM/11-as-a-user-i-want-to-be-able-to-see-the-current-activity

## Commits Checklist
- [x] commits have clear title names 
- [x] commit has descriptive message of what has changed 
- [x] more small commits than less big commits 

## Code Checklist 
- [x] code is formatted appropriately 
- [x] code meets DRY standards
- [x] no dead code (check for logs and print statements)  
- [x] no TODOS instead necessary comments 
- [x] code follows Story Squad standards for:
     - Language 
     - Framework 
     - Libraries

### CODE REVIEW

File: src/components/pages/HeadsUpDisplay/CurrentActivity.js [hunk 1]
```diff
@@ -1,117 +1,101 @@
...
+  // useEffect to call both functions on render
   useEffect(() => {
-    const timer =
-      timeRemaining > 0 &&
-      setTimeout(() => setTimeRemaining(calculateTimeRemaining()), 1000);
-    return () => clearTimeout(timer);
-  }, [timeRemaining]);
+    currDay();
+    currActivity();
+  });
```
Comment (Jtamedrano) [inline src/components/pages/HeadsUpDisplay/CurrentActivity.js:98]: Are there any dependencies? If not, might want to just add an empty dependency array

File: src/components/pages/HeadsUpDisplay/CurrentActivity.js [hunk 2]
```diff
@@ -1,117 +1,101 @@
...
+      }
+    } else if (today === 6) {
+      if (day === 5) {
+        setDay(6);
+        return;
+      } else if (day === 12) {
+        setDay(13);
+        return;
+      }
+    }
```
Comment (Jtamedrano) [inline src/components/pages/HeadsUpDisplay/CurrentActivity.js:69]: Love the execution here. There maybe a way to make it more readable, can you think of anyways to extract some of the logic into a function?

**Review by Jtamedrano — APPROVED** [review]
Overall great work here. Left a couple of comments for more syntax and extraction but good job.

File: src/components/pages/HeadsUpDisplay/CurrentActivity.js (hunk 2 above)
Comment (ngwatso) [reply src/components/pages/HeadsUpDisplay/CurrentActivity.js:69]: I have an idea how I can make this more readable, my plan was to get the feature in and then go back and reconfigure.

File: src/components/pages/HeadsUpDisplay/CurrentActivity.js (hunk 1 above)
Comment (ngwatso) [reply src/components/pages/HeadsUpDisplay/CurrentActivity.js:98]: I did have an empty array in here, but eslint kept saying it was not needed.  It is functioning like this, but I can go back and add it on next pass.

**Pull request merged**